Components/avatar/avatar.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/installGroot.html/install.css" rel="stylesheet">
+    <title>Avatar</title>
+</head>
+<body>
+    <div class="groot">
+        <div class="groot-container first">
+            <nav>
+            <img src="/images/groot.png" style="width: 4rem;height: auto; margin: 0 0.5rem "/>
+            <span style="display: inline-flex;"><a href="/" class="square">GrooT UI</a></span>
+            <div class="version"> v1.0</div>
+            </nav>
+            <div class="item-container">
+                <div class="install-ui"><a href="/installGroot.html/install.html" class="install-text"> Installation </a></div>
+                        <a href="/Components/avatar/avatar.html" class="avatar-text ">Avatars</a>
+                        <a href="/Components/alert/alert.html" class="avatar-text">Alert</a>
+                        <a href="/Components/buttons/button.html" class="avatar-text">Badges</a>
+                        <a href="/Components/badges/badges.html" class="avatar-text">Buttons</a>
+                        <a href="/Components/buttons/button.html" class="avatar-text">Cards</a>
+                        <a href="/Components/modal/modal.html" class="avatar-text">Modals</a>
+                        <a href="/Components/input/input.html" class="avatar-text">Input</a>
+                </div>
+                </div>
+        <div class="groot-container second">
+            <div class="avatar-container">
+                <span style="font-size: 2.5rem;">Avatars</span><br/>
+                <span style=><p style="font-size: 1.3rem; color: grey;border-left: 3px solid #6B7280;margin-left: 1rem;">Avatars are user profile pictures.An avatar is a picture that placed on your social media profile, role-playing game or blog to tell people who you're.It's the best way for other people to identify you, just like an ID card on the internet.</p></span>
```

Comment:
I would suggest aligning the header and the subtext below with the same margin on the left.